Taskfile.yml

```diff
@@ -35,28 +35,19 @@ tasks:
         vars:
           COMPONENT: "job-orchestration"
 
-  clean-package:
```

Comment:
Are we changing the behavior such that we can't run task:clean-package is not valid target anymore?
It makes sense for me since anyway running task package will just remove the old package anyway. but what if we want to add more fancy checks, such as make sure no existing containers are running?

Reply:
Yeah, we're removing it for now. If we have more fancy things, I'm happy to add it back.